### PR TITLE
add(subject):print-middle

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: subjects/printmiddle/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- If the number of arguments is less than 1, print (`'\n'`)
+- If the number of arguments is **even**, print the middle two arguments with a space (`' '`) between the arguments
+- If the number of arguments is **odd**, print the middle one.
+- Print (`'\n'`) at the end of the output.
+
+### Usage
+
+```console
+$ go run . | cat -e
+$
```
Comment (davidrobert99) [inline subjects/printmiddle/README.md:15]: I think it is better to remove the '$' after the output, as the function does not return that character.

File: subjects/printmiddle/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+$ go run . "Hello World!" | cat -e
+Hello World!$
+$ go run . "Hello World!" "World!" "All" | cat -e
+World!$
```
Comment (davidrobert99) [inline subjects/printmiddle/README.md:19]: For example, here it would only be 'World!'

**Review by davidrobert99 — CHANGES_REQUESTED** [review]

File: subjects/printmiddle/README.md (hunk 1 above)
Comment (Hamzaelkhatri) [reply subjects/printmiddle/README.md:15]: yes, you right, but `cat  -e` show if there's any new line after the output.
and the subject says that should add a new line after the output .

Comment (davidrobert99) [reply subjects/printmiddle/README.md:15]: oh yes, sorry for my mistake, you are right, I was running without the cat -e. Let's kept as is.

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**